containers/tgi/README.md

```diff
@@ -60,6 +60,22 @@ docker run --gpus all -ti --shm-size 1g -p 8080:8080 \
     us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-text-generation-inference-cu124.2-3.ubuntu2204.py311
 ```
 
+- **TPU**: This example showcases how to deploy a TGI server on a TPU instance using the TGI container. Note that TPU support for TGI is currently experimental and may have limitations compared to GPU deployments.us-docker.pkg.dev/deeplearning-platform-release/gcr.io/huggingface-text-generation-inference-cu124.2-3.ubuntu2204.py311
+
+
+```bash
+docker run --rm -p 8080:8080 \
+    --shm-size 16G --ipc host --privileged \
+    -e MODEL_ID=google/gemma-7b-it \
+    -e HF_TOKEN=$(cat ~/.cache/huggingface/token) \
```

Comment:
Just realised that we're missing a brief mention on where and how to retrieve and set this token, but will also do that on an upcoming PR!